src/components/App.tsx

```diff
@@ -1,10 +1,44 @@
-import { MyComponent } from "./MyComponent";
 import "./App.css";
+import { useState } from "react";
+import { Heading } from "@chakra-ui/react";
+import BoardsList from "./BoardsList";
+import Board from "./Board";
+import BoardInfo from "../interfaces/Board";
+
+const backendUrl = "https://arrangio-backend.onrender.com/";
 
 function App() {
+    const [screen, setScreen] = useState("boardsList");
```

Comment:
No type safety, instead of any string try a union type with string literals for example: `type ScreenState = "boardsList" | "board"`